roles/oneview_os_deployment_server_facts/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  role_name: oneview_os_deployment_server_facts
+  author: Asis Bagga
+  description: Ansible role to manage OneView Deployment Server Facts resources.
+  company: Hewlett Packard Enterprise
+  license: Apache
+  min_ansible_version: 2.9
+  min_python_version: 3.4.2
```

Comment:
change this to min_python_version: 2.7.9 to make it uniform

Reply:
done